src/test/java/com/google/javascript/clutz/inherit_visibility.js

```diff
@@ -0,0 +1,39 @@
+goog.module('inherit_visibility');
+
+/** @constructor */
+function Parent() {}
+
+Parent.prototype.publicImplicit = function() {};
+/** @public */
+Parent.prototype.publicExplicit = function() {};
+/** @protected */
+Parent.prototype.protected = function() {};
+/** @protected */
+Parent.prototype.protectedNoJSDoc = function() {};
+
+/** @constructor @extends {Parent} */
+function Child() {}
+
+/** @override */
+Child.prototype.publicImplicit = function() {};
+/** @override */
+Child.prototype.publicExplicit = function() {};
+/** @override */
+Child.prototype.protected = function() {};
+Child.prototype.protectedNoJSDoc = function() {};
+
+/** @constructor @extends {Child} */
+function Child2() {}
+
+// In Closure, a child class can override an implicit public property with protected.
```

Comment:
I didn't know that. Do you think that is intentional or a bug? Might be worth filing an issue on google/closure-compiler.

Reply:
Filed: https://github.com/google/closure-compiler/issues/3179